src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -86,4 +144,29 @@ interface IEigenPodManager is IPausable {
     function slasher() external view returns(ISlasher);
 
     function hasPod(address podOwner) external view returns (bool);
+
+    /// @notice returns shares of provided podOwner
+    function podOwnerShares(address podOwner) external returns (uint256);
```

Comment:
line 138 has the wrong function name, should be `getBeaconChainBlockRootFromTimestamp(uint64 timestamp)`

Reply:
so this just should be timestamp rather than blocknumber? I recall us talking about this before, just want to confirm.
I thought @Sidu28 made this change already, maybe on a different branch or something?

Reply:
looks like he just forgot to update _some_ of the usage. adding a commit to fix this, will push it shortly.